### PR TITLE
Thanos docs

### PR DESCRIPTION
**Pull request by jmreicha**
Adds some Thanos documentation.

### CODE REVIEW

File: Documentation/thanos.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Thanos and the Prometheus Operator
```
Comment (brancz) [inline Documentation/thanos.md]: Could we add a very quick thanos introduction here?

**Comment by metalmatze** [comment]
LGTM for me. Feel free to merge @brancz

**Comment by brancz** [comment]
Nice, looks good! :+1:

**Comment by brancz** [comment]
Thanks a lot for your effort @jmreicha!

**Pull request merged**